### PR TITLE
Add AVI

### PR DESCRIPTION
**Pull request by chakravarthi589**

### CODE REVIEW

**Comment by chakravarthi589** [comment]
@ajpmaclean : Please review.

**Comment by ajpmaclean** [comment]
See my comments. Are you able to read the produced file? 
Also in `AVI.md` "AVI." doesn't mean much, use a sentence to describe what the code does. 

I don't think this should go in, the C++ version is marked untested - this normally implies it is waiting for someone to fix it.

**Comment by chakravarthi589** [comment]
@ajpmaclean : I'll get back to you about the avi output file.

**Comment by chakravarthi589** [comment]
@ajpmaclean: I have tried testing the generated test.avi output file and attaching two snapshots for your reference. I changed the rectangle color from red to green and was able to see it. 
Please review. Am not sure about the correctness. seeking your input on this.
test_1
test_2

**Comment by chakravarthi589** [comment]
@ajpmaclean : Conflicts resolved. Please check now

**Comment by chakravarthi589** [comment]
@ajpmaclean : Please review the updated AVI.java and Java.md
1. Modified the program to accept filename as command line argument 
2. Made use of vtkNamedColor

File: src/Java/Untested/Video/AVI.java [hunk 1]
```diff
@@ -24,28 +25,52 @@
...
+	double boxColor[] = new double[] {255,0,0,1};
+	  
+	colors.SetColor("bkgColor", bkgColor);
+	colors.SetColor("boxColor", boxColor);
+	
+	
+	// Provide default values.
+	String file_Name = "Output.avi";
+	String File_Argument ="";
+	for(int i = 0; i < args.length; ++i)
```
Comment (ajpmaclean) [inline src/Java/Untested/Video/AVI.java]: Why do you need **File_argument**?

File: src/Java/Untested/Video/AVI.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	double bkgColor[] = new double[] {0, 0,0,1};
+	double boxColor[] = new double[] {255,0,0,1};
+	  
+	colors.SetColor("bkgColor", bkgColor);
+	colors.SetColor("boxColor", boxColor);
+	
+	
+	// Provide default values.
+	String file_Name = "Output.avi";
+	String File_Argument ="";
```
Comment (ajpmaclean) [inline src/Java/Untested/Video/AVI.java]: Use camel case (**fileName**) or underscores , all lower case (**file_name**).

Comment (chakravarthi589) [reply src/Java/Untested/Video/AVI.java]: I have made changes please review.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**